core-services/release-controller/_repos/ocp-4.8-default.repo

```diff
@@ -30,8 +30,8 @@ gpgkey = https://mirror.ops.rhcloud.com/libra/keys/RPM-GPG-KEY-redhat-release ht
 
 [rhel-server-ose]
 name = rhel-server-ose
-baseurl = https://use-mirror1.ops.rhcloud.com/enterprise/reposync/4.8/rhel-server-ose-rpms/
-          https://use-mirror2.ops.rhcloud.com/enterprise/reposync/4.8/rhel-server-ose-rpms/
+baseurl = https://use-mirror1.ops.rhcloud.com/enterprise/reposync/4.9/rhel-server-ose-rpms/
+          https://use-mirror2.ops.rhcloud.com/enterprise/reposync/4.9/rhel-server-ose-rpms/
```

Comment:
Hey @petr-muller,
Why are these 4.8 files pointing to 4.9?

Reply:
Good catch, these should have been the 4.9 files.

Reply:
Fixed